tests/cypress/views/resources.js

```diff
@@ -45,10 +45,32 @@ export const apiResources = (type, data, returnType) => {
   }
 };
 
-export const checkExistingUrls = (css1, value1, css2, value2, url) => {
+// check if this is an existing channel
+// for new channels enter the credentials, if set
+// for existing channel select the full info, which is 'path [channelNS/channelName]
+export const checkExistingUrls = (css1, value1, css2, value2, gitUrl, url) => {
```

Comment:
Took me a bit to see what `gitUrl`... maybe something like `urlFieldSelector` would be more accurate?

Reply:
Good catch, I changed it to repoFieldId